### ISSUE TITLE
screenshot links are incorrect if target directories do not exist

### ISSUE THREAD

**Issue by naruoga** [vmi/selenese-runner-java#211]
I may found a selenese-runner issue when I run a quite simple testcase with options `--screenshot-all`, `--screenshot-on-fail` and `--html-result`, but the target directories don't exist.

I run my test in Windows Server 2012 R2 64bit.

How to reproduce:

Step 1. put somewhere (for example, `%HOME%\selenese-test`) a test script like https://gist.github.com/naruoga/4cd5194745b7efc5784390cfc0a3a69c

Step 2. run it with this command:
```
> cd selenese-test
> java -jar selenese-runner.jar selenese-test.html --screenshot-all results/all --screenshot-on-fail results/fail --html-result results/html
```

NOTE: there isn't `results` directory in `%HOME%\selenese-test`

The test will run successfully.

Step 3. check the result in `results/html/TEST-selenese-test.html` and click the link to screenshots.

Expected: each links point to `results/all/selenese-test_xxxx.png`

Actual: each links point to `results/html/all/selenese-test_xxxx.png`

**Comment by vmi** [comment]
Hi,

I fixed the problem.
Please test it.

**Comment by naruoga** [comment]
@vmi I confirmed the issue is already solved with your last commit.  Thanks! 👍

**Comment by vmi** [comment]
Hi,

I released the new version 2.10.0 which fixes  this problem.
